package.json

```diff
@@ -90,6 +90,7 @@
   "dependencies": {
     "@atlaskit/button": "^16.3.6",
     "@atlaskit/css-reset": "^6.3.13",
+    "@atlaskit/menu": "^1.3.9",
```

Comment:
I'm not sure importing this solely for the loading skeleton is best, especially since the skeleton is misaligned (there's extra padding on the left).

I think if we go down the route of loading skeletons we need to build our own as there's no existing component in the design system. There's a good one we can copy from `@atlaskit/help` called `LoadingRectangle`

Reply:
Yeah, I did feel a bit icky about this one. Will use the one you suggested.

Reply:
Nice! If you need any help let me know 👍

Reply:
That looks great now! Nice work.

I just rebased on master + tweaked the last line to be a bit narrower. Hope that's ok!

![skeleton](https://user-images.githubusercontent.com/8190792/188036980-ccbb7286-c66a-483f-8543-fc33d3c9255b.png)